backend/app/services/mongo/explore.py

```diff
@@ -1363,15 +1361,15 @@ async def fetch_all_clusterings(
                 },
                 {
                     "$project": {
-                        "pca": 0,
-                        "tsne": 0,
+                        "pca": str(0),
+                        "tsne": str(0),
                     }
                 },
             ]
         )
 
     pipeline += [
-        {"$sort": {"created_at": -1}},
+        {"$sort": {"created_at": str(-1)}},
```

Comment:
Don't do that and instead change the type hint of pipeline
pipeline: List[Dict[str, object]]